modular_skyrat/modules/knives/knives.dm

```diff
@@ -0,0 +1,60 @@
+//--- BOWIE'S KNIFE (bowie knife)---\\\
+
+
+/obj/item/melee/knife/bowie
+	name = "Bowie knife"
+	desc = "A frontiersman's classic, closer to a shortsword than a knife. It boasts a full-tanged build, a brass handguard and pommel, a wicked sharp point, and a large, heavy blade, It's almost everything you could want in a knife, besides portability."
+	icon = 'modular_skyrat/modules/knives/icons/bowie.dmi'
+	icon_state = 'bowiehand'
+	lefthand_file = 'modular_skyrat/modules/knives/icons/bowie_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/knives/icons/bowie_righthand.dmi'
+	worn_icon_state = "knife"
+	force = 20 // Zoowee Momma!
+	w_class = WEIGHT_CLASS_NORMAL
+	throwforce = 15
+	wound_bonus = -15
+	bare_wound_bonus = 20
+
+  
+/obj/item/storage/belt/bowiesheath
+	name = "Bowie knife sheath"
+	desc = "A dressed-up leather sheath featuring a brass tip. It has a large pocket clip right in the center, for ease of carrying an otherwise burdensome knife."
+	icon = 'modular_skyrat/modules/knives/icons/bowiepocket.dmi'
+	icon_state = "bowiesheathe"
+	slot_flags = ITEM_SLOT_POCKETS
+	w_class = WEIGHT_CLASS_BULKY
+	resistance_flags = FLAMMABLE
+	
+/obj/item/storage/belt/bowiesheath/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_items = 1
+	STR.max_w_class = WEIGHT_CLASS_BULKY
+	STR.set_holdable(list(
+		/obj/item/melee/knife/bowie
+		))
+		
+
+/obj/item/storage/belt/bowiesheath/AltClick(mob/user)
+	if(!user.canUseTopic(src, BE_CLOSE, NO_DEXTERITY, FALSE, TRUE))
+		return
+	if(length(contents))
+		var/obj/item/I = contents[1]
```

Comment:
u cant just mark as resolved and not change this variable name to something more meaningful than "I"

Reply:
![image](https://user-images.githubusercontent.com/62520989/168927598-08e85664-8059-4363-8f9a-c987bb9922fb.png)

Reply:
![image](https://user-images.githubusercontent.com/62520989/168927620-63e5455b-9026-432d-8d2a-94aadfc2810b.png)

Reply:
i'm not prettying it up until it's actually working

Reply:
changing it to knife (and or anything else) causes it to have a nuclear meltdown about every thing in the file